.github/workflows/continuous-e2e-tests.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
       # Run for 30 minutes
       FORGE_RUNNER_DURATION_SECS: 1800
       # We expect slightly lower tps on longer timeline
-      FORGE_RUNNER_TPS_THRESHOLD: 1000
+      FORGE_RUNNER_TPS_THRESHOLD: 400
```

Comment:
On the last few runs, `fast-chaos` has a lower TPS than `chaos`, though I'm not so sure we can expect a consistent TPS from these tests nowadays.

I think to rule out infra failure for now, can we reduce the TPS threshold for chaos tests to something low like 300? Then once we fix up the tests to be more consistent, we can re-adjust the threshold accordingly

Reply:
sure can you accept?

Reply:
How can we get a graph of the tps over time? cc @geekflyer